src/h5web/visualizations/heatmap/HeatmapVis.tsx

```diff
@@ -38,18 +38,16 @@ function HeatmapVis(props: Props): JSX.Element {
   return (
     <div className={styles.root}>
       <div ref={mapAreaRef} className={styles.mapArea}>
-        {heatmapSize && (
-          <div className={styles.heatmap} style={heatmapSize}>
-            <Canvas orthographic invalidateFrameloop>
+        {heatmapStyles && (
+          <div className={styles.heatmap} style={heatmapStyles}>
+            <Canvas
+              className={styles.canvasWrapper}
```

Comment:
Just adding a classname here to simplify selectors in CSS (`.heatmap > div` becomes `.canvasWrapper`)